cosmic_ray/importing.py

```diff
@@ -61,8 +61,11 @@ def using_mutant(module_name, mutant):
     mutated AST.
 
     """
-    if module_name in sys.modules:
-        del sys.modules[module_name]
+    module_to_delete = ""
+    for parent in module_name.split('.'):
+        module_to_delete = ("%s.%s" % (module_to_delete, parent)).lstrip('.')
+        if module_to_delete in sys.modules:
+            del sys.modules[module_to_delete]
```

Comment:
This is a bit awkward and old-fashioned. I'd prefer something a bit more modern and (to me, at least) readable:
```
del_mods = set(accumulate(module_name.split('.'), '{}.{}'.format))
sys.modules = {k: v for k, v in sys.modules.items() if k not in del_mods}
```